device.yml

```diff
@@ -0,0 +1,234 @@
+%YAML 1.1
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/harp-tech/reflex-generator/main/schema/device.json
+device: SyringePump
+whoAmI: 1280
+firmwareVersion: "1.0"
+hardwareTargets: "1.0"
+registers:
+  EnableMotorDriver: &enablefunctionreg
+    address: 32
+    access: Write
+    type: U8
+    maskType: EnableFlag
+    description: Enables the motor driver.
+  EnableProtocol:
+    <<: *enablefunctionreg
+    address: 33
+    description: Enables the currently defined protocol.
+  Step:
+    address: 34
+    access: [Write, Event]
+    type: U8
+    maskType: StepState
+    description: Status of the STEP motor controller pin.
+  Direction:
+    address: 35
+    access: [Write, Event]
+    type: U8
+    maskType: DirectionState
+    description: Status of the DIR motor controller pin.
+  ForwardSwitch: &switchreg
+    address: 36
+    access: Event
+    type: U8
+    maskType: ForwardSwitchState
+    description: Status of the forward limit switch.
+  ReverseSwitch:
+    <<: *switchreg
+    address: 37
+    maskType: ReverseSwitchState
+    description: Status of the reverse limit switch.
+  DigitalInput:
+    <<: *switchreg
+    address: 38
+    maskType: DigitalInputState
+    description: Status of the digital input pin.
+  DigitalOutputSet: &output
+    address: 39
+    type: U8
+    access: Write
+    maskType: DigitalOutputState
+    description: Set the specified digital output lines.
+  DigitalOutputClear:
+    <<: *output
+    address: 40
+    description: Clear the specified digital output lines.
+  DO0Sync: &ioconfig
+    address: 41
+    type: U8
+    access: Write
+    maskType: DO0SyncConfig
+    description: Configures which signal is mimicked in the digital output 0.
+  DO1Sync:
+    << : *ioconfig
+    address: 42
+    maskType: DO1SyncConfig
+    description: Configures which signal is mimicked in the digital output 1.
+  DI0Trigger:
+    << : *ioconfig
+    address: 43
+    maskType: DI0TriggerConfig
+    description: Configures the callback function triggered when digital input is triggered.
+  Microstepping:
+    address: 44
+    type: U8
+    access: Write
+    maskType: MicrosteppingMode
```

Comment:
Consistent with the above I would call it `MicrostepMode` or simply `StepMode`.

Reply:
See previous point